helpers/ifrauBackdropService.js

```diff
@@ -1,10 +1,13 @@
+import { isFramed } from './framed.js';
+import { SlimClient } from 'ifrau/client/slim.js';
+
 let ifrauBackdropService;
 
 export async function tryGetIfrauBackdropService() {
-	if (!window.ifrauclient) return null;
+	if (!await isFramed()) return null;
```

Comment:
Since `isFramed()` has no dependence on the ifrau client, we no longer need the `window.ifrauclient` fallback check.